Week2/Yash Homework/index.js

```diff
@@ -0,0 +1,99 @@
+//Strings
+
+//1
+
+const myString = "hello,this,is,a,difficult,to,read,sentence";
+
+console.log(myString);
+
+console.log(myString.length);
+
+const array = myString.split (",");
```

Comment:
nit: no need for space between `t` and `(`

Also it is common practice  to name variables with something descriptive, e.g `words` would work here.